chevah/styleguide
# Text clean-up for all the documents.

Scope
=====

Corrections to the text and small additions and clarifications to the parts that needed it.


Why we got into this (5 whys)
=============================

Because there were a lot of small mistakes.
Because those mistakes looked a bit bad.
Because typos look bad.
Because some of the explanations needed some context.
Because life evolves and docs should just keep up.


Changes
=======

Corrected grammar issues (missing "a" and "the") and typos (additional "s" or other characters).
Added one ammendment to Rule nr. 3.
Extended the section related to Python Abstract classes with some explanations regarding the differences between Twisted and classic Python.


How to try and test the changes
===============================

reviewers: @robi-net

Please check that the proposed additions to the content fit with the general view.
